### PR TITLE
fix: Adds a missing defribilator to nukie planet

### PR DESCRIPTION
**Pull request by Brandon-Huu**
## About the PR

Added a defibrillator  to nukie planet.

## Why / Balance

Because why the fuck do the million dollar super solders not have something as basic as an AED/defibrillator. My grand mother has one attached directly to her heart but these million dollar men can't afford it? What? 

Nukies should not be rotting on nukie island.

## Media

Before:
Screenshot from 2024-05-28 18-16-38

After:
Screenshot from 2024-05-28 18-15-20

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Brandon-Huu** [comment]
Draft until I test and verify.

**Comment by Brandon-Huu** [comment]
My testing is complete. The defibrillator is there.

**Comment by Brandon-Huu** [comment]
> Fairly sure the defib is the main thing in the medical bundle. Ideally it'll be replaced at some point with a compact/combat defib and this could make more sense.

AEDs should be on walls.

**Comment by mirrorcult** [comment]
test fail is real, you saved the map incorrectly

**Comment by Brandon-Huu** [comment]
> test fail is real, you saved the map incorrectly

Do I just change this part to false?

https://github.com/space-wizards/space-station-14/pull/28362/files#diff-04846f98d89f604441b8bf7a018dcd5655751fb95940fc48f1b1688d49e19486R1634-R1635

File: Resources/Maps/Nonstations/nukieplanet.yml [hunk 1]
```diff
@@ -1632,11 +1632,14 @@ entities:
     - type: MetaData
     - type: Transform
     - type: Map
+      mapPaused: True
```
Comment (Brandon-Huu) [inline Resources/Maps/Nonstations/nukieplanet.yml:1635]: mapPaused should be set to False?

**Comment by Brandon-Huu** [comment]
Yeah, I'll just like edit the file.

I saw that the problem when I submitted the PR but didn't think it was a problem since I did other map PRs and I followed the same steps for this one.

**Comment by Brandon-Huu** [comment]
I have another PR I want to do or this map after this is merged so, I hope I fixed the map being paused issue

**Comment by metalgearsloth** [comment]
Looks like this needs redoing from what I can see.

**Comment by Brandon-Huu** [comment]
> Looks like this needs redoing from what I can see.

How so?

**Comment by Brandon-Huu** [comment]
Did not mean to close this

**Pull request merged**